FSharp.AutoComplete/Program.fs

```diff
@@ -607,6 +611,21 @@ module internal Main =
 
             main state
 
+
+          | Methods ->
+            let meth = tyRes.GetMethods(line, col, lineStr)
```

Comment:
I think at this point we should first scan back using [this code](https://github.com/fsharp/xamarin-monodevelop-fsharp-addin/blob/master/MonoDevelop.FSharpBinding/FSharpTextEditorCompletion.fs#L318). An editor should be able to give the current cursor position with some parameters already filled in.